user_guide/src/howcani/combining_data/concatenating.md

```diff
@@ -53,6 +53,6 @@ Diagonal concatenation generates nulls when the column names do not overlap.
 
 ## Rechunking
 
-Before a concatenation we have two dataframes `df1` and `df2`. In memory each of these is a separate Arrow `Table`. By default these `Tables` are copied to a new combined table during the concatenatation - this is known as **rechunking**. Rechunking makes further operations on the concatenated `DataFrame` faster.
+Before a concatenation we have two dataframes `df1` and `df2`. Each column in `df1` and `df2` is in one or more chunks in memory. By default, during concatenation the chunks in each column are copied to a single new chunk - this is known as **rechunking**. Rechunking makes the join slower but further operations on the concatenated `DataFrame` faster.
```

Comment:
Maybe don't call it `join`? 

~Rechunking makes the join slower but further operations on the concatenated `DataFrame` faster.'~ 

Rechunking is an expensive operation, but is often worth it because future operations will be faster.

Reply:
Done